### DISCUSSION TITLE
Restoring an order that was canceled by mistake

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#2988] [Ideas]
I recently faced the situation when the manager of a store canceled several orders by mistake. I manually edited certain records in the database and returned the orders to their initial state. Is there an extension for such situations or could we add a facility to restore a canceled order in OpenMage?

**Comment by luigifab** [comment]
I have [this method](https://github.com/kyrena/openmage-paymentmax/blob/cd32e42e2fa9538befc043e0f5cb098db0b43f86/src/app/code/community/Kyrena/Paymentmax/Model/Payment.php#L255), based on: https://magento.stackexchange.com/q/7254/101197
Not sure if all cases are covered.

**Comment by flyingdog88** [comment]
This discussion here will cover that since you can edit and uncanceled orders please vote 
https://github.com/OpenMage/magento-lts/discussions/2918****

**Comment by addison74** [comment]
This could be a good start to implement this feature https://github.com/mamocbv/UnCancel.
